invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -24,6 +24,9 @@ body {
       &.inline {
         display: inline-block !important;
       }
+      &.flex {
```

Comment:
Instead of adding it everywhere would it make sense to just add important to `flex` class [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides#L486) ping @kpsherva

Reply:
> LGTM!
> 
> In the screenshot you shared of the tablet view, I would try to fix the alignment between the button and the searchbar, I think there is enough space to keep the button with the same width as in the computer ![picture1](https://user-images.githubusercontent.com/15194802/176141384-faa743ab-2a23-4345-ba94-3ababba3ff24.png)

I can/will patch it a bit for the english language. But, using fixed widths in grid, as we are doing for every view, I cannot guarantee it will work with all locales.